hasjob/views/index.py

```diff
@@ -574,7 +574,7 @@ def filterset_view(name):
         abort(404)
     return index(filters=filterset.to_filters(translate_geonameids=True),
         query_string=filterset.keywords,
-        filterset=filterset)
+        filterset=filterset, title=filterset.title)
```

Comment:
@vidya-ram maybe put these 2 on second line beside `query_string`.

Reply:
For style consistency, use one parameter per line OR as many will fit on a single line.